msteams-platform/tabs/how-to/using-teams-client-sdk.md

```diff
@@ -1,72 +1,469 @@
+
+##### Deep links out of your app
+
+For deep links from your app to various areas of its current host, use the strongly typed APIs provided by the TeamsJS SDK. For example, use the *Calendar* capability to open a scheduling dialog or calendar item from your app.
+
+For deep links from your app to other apps running in the same host, use `pages.navigateToApp`.
+
+For any other external deep linking scenarios, you can use `app.openLink`, which provides similar functionality to the now deprecated (starting in TeamsJS v.2.0) *executeDeepLink* API.
+
+#### Dialogs
+
+Starting with version 2.0 of TeamsJS, the Teams platform concept of [task module](../../task-modules-and-cards/what-are-task-modules.md) has been renamed to *dialog* for better consistency with existing concepts across the Microsoft 365 developer ecosystem. Accordingly, the `tasks` namespace has been deprecated in favor of the new `dialog` namespace.
+
+This new dialog capability is split into a main capability (`dialog`) for supporting HTML-based dialogs and a subcapability, `dialog.bot`, for bot-based dialog development.
+
+The dialog capability doesn't yet support adaptive card based dialogs. Adaptive-card based dialogs still need to be invoked using `tasks.startTask()`.
+
+The `dialog.open` function currently only works for opening HTMl-based dialogs, and it returns a callback function (`PostMessageChannel`) you can use to pass messages (`ChildAppWindow.postMessage`) to the newly opened dialog.  `dialog.open` returns a callback (rather than a Promise) because it doesn't require app execution to pause waiting for the dialog to close (thus providing more flexibility for various user interaction patterns).
+
+## What's new in TeamsJS version 2.0
+
+There are two significant changes between TeamsJS v.1.12 and v.2.0 and later:
+
+* [**Callback functions now return Promise objects.**](#callbacks-converted-to-promises) Most functions with callback parameters in TeamsJS v.1.12 have been modernized to return a JavaScript [Promise](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise) object for improved handling of asynchronous operations and code readability.
+
+* [**APIs are now organized into *capabilities*.**](#apis-organized-into-capabilities) You can think of capabilities as logical groupings of APIs that provide similar functionality, such as `authentication`, `dialog`, `chat`, and `calendar`. Each namespace represents a separate capability.
+
+> [!TIP]
+> You can use the [Teams Toolkit extension](https://aka.ms/teams-toolkit) for Microsoft Visual Studio Code to simplify the [TeamsJS v.2.0 update process](#updating-to-the-teams-client-sdk-v200) for an existing Teams app.
+
+### Backwards compatibility
+
+Once you start referencing `@microsoft/teams-js@2.0.0-beta.1` (or later) from an existing Teams app, you'll see deprecation warnings for any code calling APIs that have changed.
```

Comment:
Is this intentionally referencing a beta version or should it be updated to reference `@microsoft/teams-js@2.0.0`?

Reply:
Especially since all the deprecation warnings in the SDK have been updated to say they are deprecated as of 2.0.0.